ui/selectors/selectors.js

```diff
@@ -125,7 +126,7 @@ export function getMetaMetricsId(state) {
 }
 
 export function isCurrentProviderCustom(state) {
-  const provider = getProvider(state);
+  const provider = getProviderConfig(state);
```

Comment:
Should we also rename this variable to `providerConfig`?

Reply:
This will be done in the PR following this one. The `provider` => `providerConfig` rename ended up being pretty large so I did it in three parts:
* Get the selector in-place (this PR)
* Expand usage of the selector
* Rename